### PR TITLE
Changes for editing background, and restricting asset selection

### PR DESCRIPTION
**Pull request by amoliksingh**
Changes for editing background, and restricting asset selection

### CODE REVIEW

File: src/components/components/Component.js [hunk 1]
```diff
@@ -28,7 +28,10 @@ export default class Component extends React.Component {
     this.state = {
       entity: this.props.entity,
       name: this.props.name,
-      nameList: []
+      nameList: [],
```
Comment (ahmedhamodi) [inline src/components/components/Component.js:31]: Do we still need nameList or do the two subsequent lists (objectList and backgroundList) cover everything?

Comment (amoliksingh) [reply src/components/components/Component.js:31]: Still needed due to the value={this.state.nameList.filter(option => option.value == componentData.data)} line in the Select component returned for gltf-model

**Comment by ahmedhamodi** [comment]
Can you add a loom video showing this working? I don't have aframe-inspector setup locally so can't really test. Can also setup a time between us to do a live demo if that is easier for you

**Comment by amoliksingh** [comment]
> Can you add a loom video showing this working? I don't have aframe-inspector setup locally so can't really test. Can also setup a time between us to do a live demo if that is easier for you

https://www.loom.com/share/b48249e8f3684a68a7bab743adc9fc40

**Comment by ahmedhamodi** [comment]
Thank you Amolik! Video looks good.

@jaydhulia any thoughts on how this is gonna impact the navigation mesh?

**Review by ahmedhamodi — APPROVED** [review]

**Pull request merged**